apps/dashboard/src/components/workflow-editor/steps/in-app/in-app-editor-preview.tsx

```diff
@@ -48,10 +54,9 @@ export const InAppEditorPreview = () => {
 
         <CollapsibleContent className="flex flex-col gap-2 rounded-lg p-1">
           <Editor
```

Comment:
<!-- Suggestions propose improvements to the current subject. It's important to be explicit and clear on what is being suggested and why it is an improvement. Consider using patches and the blocking or non-blocking decorations to further communicate your intent.  -->
✏ **suggestion:** This can be wrapped inside the formField formcontrol, you wont need the onchange, value props‏